Sources/ProjectDescription/Environment.swift

```diff
@@ -10,6 +10,7 @@ public struct Environment {
         /// - Parameters:
         ///   - default: default String value to be returned
         /// - Returns: String
+        @available(*, deprecated, message: "Was moved to Optional extension. Remove question mark to fix.")
```

Comment:
I'd rephrase it, user does not have to really know the implementation details - something like: `Environment variables as optionals are deprecated. Remove question mark to fix deprecation warning.`